src/h5web/vis-packs/core/heatmap/HeatmapVis.tsx

```diff
@@ -86,7 +86,8 @@ function HeatmapVis(props: Props) {
         axisOffsets={axisOffsets}
         aspectRatio={layout === 'contain' ? cols / rows : undefined}
       >
-        <HeatmapAxisSystem
+        <AxisSystem
+          visRatio={layout === 'cover' ? cols / rows : undefined}
```

Comment:
Using a less specific term than `imageRatio`, as now we pass this ratio directly to `AxisSystem` (instead of adding an extra layer with `HeatmapAxisSystem`).